cardano-cli/src/Cardano/CLI/EraBased/Run/CreateTestnetData.hs

```diff
@@ -229,13 +233,18 @@ runGenesisCreateTestNetDataCmd Cmd.GenesisCreateTestNetDataCmdArgs
   poolParams <- forM [ 1 .. numPools ] $ \index -> do
     let poolDir = poolsDir </> ("pool" <> show index)
 
-    createPoolCredentials keyOutputFormat poolDir
+    createPoolCredentials desiredKeyOutputFormat poolDir
     buildPoolParams networkId poolDir Nothing (fromMaybe mempty mayStakePoolRelays)
 
   writeREADME poolsDir poolsREADME
 
   -- Stake delegators
-  let (delegsPerPool, delegsRemaining) = divMod numStakeDelegators numPools
+  forM_ [ 1 .. numStakeDelegators] $ \index -> do
+    case stakeDelegatorsKind of
```

Comment:
Why not move this into the following `case stakeDelegatorsKind of` instead of casing twice?

Reply:
Done :+1: